pkg/bundle/resources.go

```diff
@@ -120,7 +120,8 @@ func chartURL(location *fleet.HelmOptions, auth Auth) (string, error) {
 		pool.AppendCertsFromPEM(auth.CABundle)
 		transport := http.DefaultTransport.(*http.Transport).Clone()
 		transport.TLSClientConfig = &tls.Config{
-			RootCAs: pool,
+			RootCAs:    pool,
+			MinVersion: tls.VersionTLS12,
```

Comment:
I hope all clients are ok with that. Who uses that endpoint, k8s kubelet or fleet itself?

Reply:
As discussed, TLS was introduced 2008, is also the Minimum in modern browsers like Firefox without know issues, so it should be fine.
We will document it in the next release notes though.